projects/knora-ui/src/lib/viewer/values/boolean-value/boolean-value.component.spec.ts

```diff
@@ -17,7 +18,7 @@ import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 })
 class TestHostDisplayValueComponent implements OnInit {
 
-  @ViewChild('booleanVal', { static: false }) booleanValueComponent: BooleanValueComponent;
```

Comment:
@flavens This defaults to false anyway so `{ static: false }` is not needed, right?

Reply:
Yes, I will check the documentation and update them

Reply:
ok, then I think removing it is fine.

Reply:
I checked the docs and the sentence " Defaults to false." is new in Angular 9. So I guess it was needed before Angular 9.

Reply:
In Angular 8, the recommendation during the upgrade from 7 to 8 was to add { static: false } for ViewChild in anticipation of Angular 9. It was only a temporary fix.

Reply:
Ok, got it. Thanks for the explanation!